File: filmy-app/app/src/main/java/rk/entertainment/filmy/FilmyApplication.java
package rk.entertainment.filmy;

import androidx.multidex.MultiDex;
import androidx.multidex.MultiDexApplication;

public class FilmyApplication extends MultiDexApplication {

    @Override
    public void onCreate() {
        super.onCreate();
        MultiDex.install(this);
    }
}

File: filmy-app/app/src/main/java/rk/entertainment/filmy/FilmyApplication.kt
package rk.entertainment.filmy

import android.content.Context
import androidx.multidex.MultiDex
import androidx.multidex.MultiDexApplication
import dagger.hilt.android.HiltAndroidApp

@HiltAndroidApp
class FilmyApplication : MultiDexApplication() {

    companion object {
        lateinit var appContext: Context
    }

    override fun onCreate() {
        super.onCreate()
        MultiDex.install(this)
        appContext = this
    }
}